docs/sdk/wallets/minting-methods/mint-via-social.md

```diff
@@ -0,0 +1,116 @@
+
+If you are using Lit Relay Server, you will need to request an API key [here](https://forms.gle/RNZYtGYTY9BcD9MEA).
+
+:::
+
+### Handling the Redirect
+
+At the `redirectUri` specified when initializing the providers, call `handleSignInRedirect`. You can also use `isSignInRedirect` method to check if the app is in the redirect state or not.
+
+```javascript
+import { LitAuthClient, isSignInRedirect } from '@lit-protocol/lit-auth-client';
+import { AuthMethodScope, ProviderType } from '@lit-protocol/constants';
+
+async function handleRedirect() {
+  // Check if app has been redirected from Lit login server
+  if (isSignInRedirect(redirectUri)) {
+    // Get the provider that was used to sign in
+    const provider = provider = litAuthClient.getProvider(
+      ProviderType.Google,
+    );
+    // Get auth method object that has the OAuth token from redirect callback
+    const authMethod: AuthMethod = await provider.authenticate();
+    // -- setting scope for the auth method
+    // <https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scopes>
+    const options = {
+        permittedAuthMethodScopes: [[AuthMethodScope.SignAnything]],
+    };
+    // Mint PKP using the auth method
+    const mintTx = await provider.mintPKPThroughRelayer(
+        authMethod,
+        options
+    );
+    // Fetch PKPs associated with the authenticated social account
+    const pkps = await provider.fetchPKPsThroughRelayer(authMethod);
+    return pkps;
+  }
+}
+```
+
+The provider's `authenticate` method validates the URL parameters returned from Lit's login server after a successful login, and then returns an `AuthMethod` object containing the OAuth token.
+
+With the `AuthMethod` object, you can mint and fetch PKPs associated with the authenticated social account. View the available methods in the [API docs](https://js-sdk.litprotocol.com/modules/lit_auth_client_src.html).
+
+### Generating `SessionSigs`
+
+After successfully authenticating with a social login provider, you can generate `SessionSigs` using the provider's `getSessionSigs` method. The `getSessionSigs` method takes in an `AuthMethod` object, optional `LitNodeClient` object, a PKP public key, and other session-specific arguments in `SessionSigsParams` object such as `resourceAbilityRequests` and `chain`. View the [API Docs](https://js-sdk.litprotocol.com/interfaces/types_src.BaseProviderSessionSigsParams.html).
+
+```javascript
+// Get session signatures for the given PKP public key and auth method
+const sessionSigs = await provider.getSessionSigs({
+  authMethod: '<AuthMethod object returned from authenticate()>',
+  pkpPublicKey: pkps[0].publicKey, // Note, an AuthMethod can own more than one PKP
+  sessionSigsParams: {
+    chain: 'ethereum',
+    resourceAbilityRequests: [{
+        resource: litResource,
+        ability: LitAbility.AccessControlConditionDecryption
+      }
+    ],
+  },
+});
+```
+
+
+## Email / SMS (OTP)
```

Comment:
can we include any of this code here? The section it titled 'OTP / SMS' but none of this is included here

Reply:
Done @a1ttech :)